packages/shared/forks/ReactFeatureFlags.native-fb-dynamic.js

```diff
@@ -15,10 +15,6 @@ import typeof * as DynamicFlagsType from 'ReactNativeInternalFeatureFlags';
 //
 // Use __VARIANT__ to simulate a GK. The tests will be run twice: once
 // with the __VARIANT__ set to `true`, and once set to `false`.
-//
-// TODO: __VARIANT__ isn't supported for React Native flags yet. You can set the
-// flag here but it won't be set to `true` in any of our test runs. Need to
-// update the test configuration.
 
```

Comment:
How does the import  here not just a circular dependency? The `ReactNativeInternalFeatureFlags` import should suffice, since that's defined in flow/xplat.js